dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/deduplication/DeduplicationServiceTest.java

```diff
@@ -163,6 +168,13 @@ void shouldBeAutoMergeable()
           PotentialDuplicateForbiddenException,
           ForbiddenException,
           NotFoundException {
+    SystemUser actingUser = new SystemUser();
+    Authentication authentication =
+        new UsernamePasswordAuthenticationToken(actingUser, "", actingUser.getAuthorities());
+    SecurityContext context = SecurityContextHolder.createEmptyContext();
+    context.setAuthentication(authentication);
+    SecurityContextHolder.setContext(context);
```

Comment:
These lines look familiar :D I think this is the static helper again I pointed out or one that also should exist.